pom.xml

```
@@ -73,8 +74,8 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.462.x</artifactId>
-                <version>3221.ve8f7b_fdd149d</version>
+                <artifactId>bom-weekly</artifactId>
```

Per https://github.com/jenkinsci/bom?tab=readme-ov-file#depending-on-bom-weekly

```suggestion
                <artifactId>bom-2.462.x</artifactId>
```